packages/neon/neon/lib/src/models/app_implementation.dart

```diff
@@ -46,13 +67,25 @@ abstract class AppImplementation<T extends Bloc, R extends NextcloudAppOptions>
     final core.OcsGetCapabilitiesResponseApplicationJson_Ocs_Data capabilities,
   );
 
+  /// Cache for all blocs.
+  ///
+  /// To access a bloc use [getBloc] instead.
   final blocsCache = AccountCache<T>();
```

Comment:
Shouldn't this be private?

Reply:
We can think about this in a followup but we need it public for now so the both the AppsBloc and AccountsBloc can dispose dangling blocs from the cache.